src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -181,12 +236,25 @@ public void WriteCritical(string message, params object[] args)
             }
         }
 
+        /// <summary>
+        /// Writes an exception log.
+        /// </summary>
+        /// <param name="level"><see cref="EventLevel"/></param>
+        /// <param name="innerException"><see cref="Exception"/></param>
+        /// <param name="message">The log message</param>
         [NonEvent]
         public void Write(EventLevel level, Exception innerException, string message)
         {
             Write(level, innerException, message, null);
         }
 
+        /// <summary>
+        /// Writes an exception log without generating an event.
```

Comment:
should be same "summary" as a above. Only difference is 'formatting parameters' are passed.